static_report/src/components/shared/Footer.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Flex, Text, Link, Image, Divider } from '@chakra-ui/react';
+
+export function Footer() {
+  return (
+    <Flex gap={8} my={8}>
+      <Flex gap={3}>
+        <Flex alignItems="center" gap={3}>
+          <Link href="https://github.com/InfuseAI/piperider" isExternal>
+            <Image
+              src="GitHub/GitHub-Logo.png"
```

Comment:
Hmm, should we also consider a `svg` for github logo?

Reply:
Haha, I download the logo from the official site, but they didn't provide SVG files, let me download it from other sites.